### ISSUE TITLE
track POSIX unlink calls

### ISSUE THREAD

**Issue by shanedsnyder** [darshan-hpc/darshan#724]
Should we add a new wrapper/counter for `POSIX_UNLINKS` to the POSIX module? I'm a little surprised we don't have one yet, but maybe its just never something our users or our own use cases have been interested in. This will probably be helpful for understanding Python frameworks, ML frameworks, etc. that may use lots of temporary files.

**Comment by carns** [comment]
Yeah, that seems like a good idea.  Would be neat for the report to be able to indicate how many ephemeral files were generated...

**Comment by shanedsnyder** [comment]
`ftruncate()` probably another good candidate to add support for. Reported by John Bauer.
